python/py-cleo/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-cleo
+version             0.7.6
+revision            0
+categories-append   devel
+platforms           darwin
+license             MIT
+supported_archs     noarch
+
+python.versions     37 38
+
+maintainers         {gmail.com:davidgilman1 @dgilman} openmaintainer
+
+description         \
+    Cleo allows you to create beautiful and testable \
+    command-line interfaces.
+
+long_description    \
+    Create beautiful and testable command-line interfaces. \
+    \
+    Cleo is mostly a higher level wrapper for CliKit, so a \
+    lot of the components and utilities comes from it. \
+    Refer to its documentation for more information.
+
```

Comment:
I personally think the indentation of ```description``` and ```long_description``` looks weird... What's wrong with just having it all aligned - it's still less than 80 characters, right?